wit/wit.py

```diff
@@ -127,6 +127,18 @@ def interactive(self, context=None):
                 return
             print(self.message(message, context))
 
+    def samples(self, text, entities):
```

Comment:
could we pass in an array of samples instead?

(self, samples)

the current implementation makes it so you have to send one request per sample